src/graphql/query/myKiva.graphql

```diff
@@ -35,40 +83,26 @@ query myKivaQuery {
       filter: {
         category: loan
       }
```

Comment:
The default limit is 20?

Reply:
The user still sees their repayments like expected?

Reply:
Yeah, for both cases, transactions and loans.

Reply:
Repayments are merged with updates and still working as expected.

Reply:
I know we've been talking about needing a filter for transactions, but what are the current limitations of this refactor?

Reply:
For sure!
Probably a temporary if we include new filter in transactions query soon.

Reply:
Yeah definitely mention the temporary nature, maybe a month-ish?